subprojects/gradle/code-ownership/src/gradleTest/kotlin/com/avito/android/StrictOwnershipTest.kt

```diff
@@ -49,7 +49,7 @@ internal class StrictOwnershipTest {
                                     |def speed = new Owner() { }
                                     |
                                     |ownership {
-                                    |    owners = [speed]
+                                    |    owners(speed)
                                     |}
```

Comment:
Now the error message is correct only for kts.
If the client uses groovy DSL, it will be confusing.

There are two options:
- Provide the same API for Groovy DSL
- Show another error message